spec/bulk_data_test_kit/bulk_data_group_export_cancel_stu1_spec.rb

```diff
@@ -41,7 +41,8 @@ def run(runnable, inputs = {})
           url :bulk_server_url
         end
 
-        input :bulk_server_url, :bearer_token, :group_id
+        input :bulk_server_url, :group_id
```

Comment:
These inputs need to be defined on the tests. We shouldn't have to define them in the specs.